src/auth/auth.controller.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  Body,
+  Controller,
+  Post,
+  HttpCode,
+  HttpStatus,
+  Get,
+} from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { BaseUser } from 'src/user/dto/base-user.dto';
+import { Public } from './public-strategy';
+
+@Controller('auth')
+@ApiTags('auth')
```

Comment:
(praise): I see you've been reading the Nest documentation for other ways to enhance your API. Great job!